File: sample/TestAS2Spring/src/com/azri/as4j/model/DataVo.java
package com.azri.as4j.model;


/**
 * 数据
 * @author 破晓
 *
 */
public class DataVo {
	/** 通信ID */
	private String ACUID = "";
	/** 服务名称 */
	private String serviceName;
	/** 调用的方法名称 */
	private String methodName;
	/** 参数列表 */
	private Object[] params;
	
	/** 执行状态（error/success） */
	private String resultStatus;
	/** 返回值 */
	private Object result;
	
	/** 异常信息 */
	private String errorMessage;
	
	
	public String getResultStatus() {
		return resultStatus;
	}
	public void setResultStatus(String resultStatus) {
		this.resultStatus = resultStatus;
	}
	public Object getResult() {
		return result;
	}
	public void setResult(Object result) {
		this.result = result;
	}
	public String getErrorMessage() {
		return errorMessage;
	}
	public void setErrorMessage(String errorMessage) {
		this.errorMessage = errorMessage;
	}
	public String getServiceName() {
		return serviceName;
	}
	public void setServiceName(String serviceName) {
		this.serviceName = serviceName;
	}
	public String getMethodName() {
		return methodName;
	}
	public void setMethodName(String methodName) {
		this.methodName = methodName;
	}
	public Object[] getParams() {
		return params;
	}
	public void setParams(Object[] params) {
		this.params = params;
	}
	public String getACUID() {
		return ACUID;
	}
	public void setACUID(String aCUID) {
		ACUID = aCUID;
	}
}

File: AS2DotNet/WEB/App_Code/as2DotNet/DataVO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace as2DotNet
{
    /// <summary>
    ///DataVO 的摘要说明
    /// </summary>
    public class DataVO
    {
        /** 通信ID */
        private String _ACUID = "";
        private String _serviceName;
        private String _methodName;
        private Object[] _param;
        private String _resultStatus;
        private Object _result;
        private String _errorMessage;

        public String ACUID
        {
            get { return _ACUID; }
            set { _ACUID = value; }
        }

        public String ServiceName
        {
            get { return _serviceName; }
            set { _serviceName = value; }
        }

        public String MethodName
        {
            get { return _methodName; }
            set { _methodName = value; }
        }

        public Object[] Param
        {
            get { return _param; }
            set { _param = value; }
        }

        public String ResultStatus
        {
            get { return _resultStatus; }
            set { _resultStatus = value; }
        }

        public Object Result
        {
            get { return _result; }
            set { _result = value; }
        }

        public String ErrorMessage
        {
            get { return _errorMessage; }
            set { _errorMessage = value; }
        }
    }
}

